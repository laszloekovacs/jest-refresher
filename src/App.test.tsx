import React from 'react';
import renderer from 'react-test-renderer';
import App from './App'


it('renders out the word app', ()=> {
    const component = renderer.create(<App></App>)

    let tree = component.toJSON();

    expect(tree).toMatchSnapshot()

})
