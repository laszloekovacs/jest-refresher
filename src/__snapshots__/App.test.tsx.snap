// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders our the word app 1`] = `
<div>
  Notinhg
</div>
`;
